internal/store/serverservice.go

```diff
@@ -43,7 +43,7 @@ var (
 	ErrServerserviceAttribute = errors.New("error in serverservice attribute")
 
 	// ServerserviceConditionsNSFmtStr attribute namespace format string value for server condition attributes.
-	ServerserviceConditionsNSFmtStr = "sh.hollow.condition.%s"
+	ServerserviceConditionsNSFmtStr = "sh.hollow.rctypes.%s"
```

Comment:
The name of the import should not be leaking into the NATS subject name. An outside observer expects the “thing” we’re referring to as a “condition”. The token we use to import this is an internal implementation detail.

Reply:
thanks, fixed